src/event-stream/event-server.ts

```diff
@@ -41,6 +46,10 @@ async function handleMempoolTxsMessage(rawTxs: string[], db: DataStore): Promise
     };
   });
   for (const tx of decodedTxs) {
+    // Send update to websocket subscribers
+    txSubscribers
```

Comment:
(looks like this comment was removed after a rebase) the websocket code here should be removed, it's more appropriately handled in the db event emitter hooks

Reply:
my bad, thought i'd already done this! removing...